chi/lease.py

```diff
@@ -42,6 +46,7 @@ class ErrorParsers:
 
 BLAZAR_TIME_FORMAT = "%Y-%m-%d %H:%M"
 NODE_TYPES = {
+    "compute_haswell",
```

Comment:
Thanks for catching this, though this list is really outdated, and hasn't been updated in a long time. It's a bit complicated since now there are so many different sites with different types of nodes.